Modules/Core/SpatialObjects/include/itkEllipseSpatialObject.h

```diff
@@ -54,65 +53,61 @@ class ITK_TEMPLATE_EXPORT EllipseSpatialObject:
   using PointContainerPointer = SmartPointer< PointContainerType >;
 
   using ArrayType = FixedArray< double, TDimension >;
-  static constexpr unsigned int NumberOfDimension = TDimension;
+  static constexpr unsigned int ObjectDimension = TDimension;
 
   itkNewMacro(Self);
   itkTypeMacro(EllipseSpatialObject, SpatialObject);
 
   /** Set all radii to the same radius value.  Each radius is
    *  half the length of one axis of the ellipse.  */
-  void SetRadius(double radius);
+  void SetRadiusInObjectSpace(double radius)
```

Comment:
2 comments concerning this function:
* Since the function was renamed, it would be important to also update the documentation and highlight that there are 2  spaces: World and Object.

* In `Set` functions, the input value should be compared to the current value. It should only update the current value and call `Modified` if the new value is different.

Reply:
This (and others) should be updated to use ITK Set Macros.   Thanks for catching it.    Went through a couple of iterations before deciding the relevant space (object or world) should be explicit in member functions and variables.   These functions are remnants of a best forgotten version :)